docs/csharp/language-reference/operators/namespace-alias-qualifier.md

```diff
@@ -1,45 +1,64 @@
 ---
 title: ":: operator - C# reference"
 ms.custom: seodec18
-ms.date: 07/20/2015
+ms.date: 08/09/2019
 f1_keywords: 
   - "::_CSharpKeyword"
+  - "global_CSharpKeyword"
 helpviewer_keywords: 
   - ":: operator [C#]"
-  - "namespaces [C#], :: operator"
-  - "namespace alias qualifier operator (::) [C#]"
+  - "namespace alias qualifier [C#]"
+  - "namespace [C#]"
+  - "global keyword [C#]"
 ms.assetid: 698b5a73-85cf-4e0e-9e8e-6496887f8527
 ---
 # :: operator (C# reference)
 
-The namespace alias qualifier (`::`) is used to look up identifiers. It is always positioned between two identifiers, as in this example:
+Use the namespace alias qualifier `::` to access members of an aliased namespace. You use the `::` qualifier between two identifiers. The left-hand identifier can be any of the following aliases:
 
-[!code-csharp[csRefOperators#27](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefOperators/CS/csrefOperators.cs#27)]
+- A namespace alias created with the [using alias directive](../keywords/using-directive.md):
+  
+  ```csharp
+  using forwinforms = System.Drawing;
+  using forwpf = System.Windows;
+  
+  public class Converters
+  {
+      public static forwpf::Point Convert(forwinforms::Point point) => new forwpf::Point(point.X, point.Y);
+  }
+  ```
 
-The `::` operator can also be used with a *using alias directive*:
+- An [extern alias](../keywords/extern-alias.md).
+- The `global` alias, which is the global namespace alias. The global namespace is the namespace that contains namespaces and types that are not declared inside a named namespace. When used with the `::` qualifier, the `global` alias always references the global namespace, even if there is the user-defined `global` namespace alias.
+  
+  The following example uses the `global` alias to access the .NET <xref:System> namespace, which is a member of the global namespace. Without the `global` alias, the user-defined `System` namespace, which is a member of the `MyCompany.MyProduct` namespace, would be accessed:
```

Comment:
With that example in this article, I suggest to remove and redirect the following programming guide article:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/namespaces/how-to-use-the-global-namespace-alias

Reply:
I agree. Do you want to add it to this PR, or make a separate PR?

Reply:
@BillWagner I prefer to make it as a separate PR. I'll resolve the merge conflict with TOC and then it's ready to ship.